src/Context.php

```diff
@@ -87,15 +89,15 @@ private function signingKey()
         }
     }
 
-    /**
-     * @return HmacAlgorithm
-     *
-     * @throws Exception
-     */
-    private function algorithm()
-    {
-        return Algorithm::create($this->algorithmName);
-    }
+    // /**
+    //  * @return HmacAlgorithm
+    //  *
+    //  * @throws Exception
+    //  */
+    // private function algorithm()
+    // {
+    //     return Algorithm::create($this->algorithmName);
+    // }
```

Comment:
remove code instead of commenting out

Reply:
Fixed in 665baabfcb62596b838b3c852613f45b81be700e